README.md

```diff
@@ -1,2 +1,91 @@
-# Hello-Microverse
-This is the first Microverse project on the first week.
+<a name="readme-top"></a>
+
+<div align="center">
+  <h1> Hello Microverse </h1>
+  <br/>
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+**[Hello Microverse]** is the first introductory Project in the first week of the program.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+<details>
```

Comment:
Good job including a readme.md file in the repository. But currently, the readme is not properly written since there are important sections that were not added `(e.g key features, Live demo, prerequisite, usage, install, run test, deployment, show support, and the future features section)`. Please add the recommended [Microverse](https://github.com/microverseinc/readme-template) readme template the way it was provided in order to have a professional documentation.

**NOTE : ** All sections of this readme are important and should not be removed as you have done. The only section that is optional is the FAQ section. You can checkout this example of a good [readme](https://github.com/sunga12/My-Portfolio/tree/setup-mobile-first)